steps/genymotion-saas-install-gmsaas-cli/0.0.1/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Genymotion SaaS install gmsaas cli
+summary: |
+  Install gmsaas,  command line interface to rule all your virtual devices hosted in Genymotion SaaS solution.
+description: |
+  This step installs gmsaas cli, it allaows user to rule all your virtual devices hosted in Genymotion SaaS solution without opening an Internet browser.
+website: https://github.com/thomascarpentier/bitrise-step-genymotion-saas-install-gmsaas-cli
+source_code_url: https://github.com/thomascarpentier/bitrise-step-genymotion-saas-install-gmsaas-cli
+support_url: https://github.com/thomascarpentier/bitrise-step-genymotion-saas-install-gmsaas-cli/issues
+published_at: 2023-01-25T11:57:35.878313175+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-saas-install-gmsaas-cli.git
+  commit: 2ca25eaf9c778cfe35d3a31bc072b24820ead71c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
```

Comment:
nit: Xamarin platform is now deprecated, feel free to remove it from this list

Reply:
Hi @ofalvai, thanks for the review. I've updated all files.